netsim/tools/nuts/tests/test_interfaces.j2

```diff
@@ -0,0 +1,14 @@
+- test_class: TestNapalmInterfaces
+  test_data:
+{% for nn,nd in nodes.items()
+     if nd.ansible_connection|default('') in ['ssh','paramiko','network_cli']
```

Comment:
You could replace this with the checks for `node.device` belonging to one of the inventory schema. Also, _netlab_ is pretty unhappy if a node has no management address, so it's safe to assume it has one.